core/iwasm/include/wasm_export.h

```diff
@@ -252,6 +252,11 @@ typedef struct LoadArgs {
     const strings), making it possible to free the wasm binary buffer after
     loading. */
     bool wasm_binary_freeable;
+
+    /* false by default, if true, don't resolve the symbols yet. The
+       wasm_runtime_load_ex has to be followed by a wasm_runtime_resolve_symbols
+       call */
+    bool no_resolve;
```

Comment:
Should also add the field in the struct LoadArgs in wasm_c_api.h, note that LoadArgs is defined in both wasm_export.h and wasm_c_api.h and is controlled by macro LOAD_ARGS_OPTION_DEFINED.

Reply:
Ah. I'd missed that. Thanks. Fixed now.